src/app/Http/Controllers/Operations/FetchOperation.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Controllers\Operations;
+
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Route;
+
+trait FetchOperation
+{
+    /**
+     * Define which routes are needed for this operation.
+     *
+     * @param string $segment    Name of the current entity (singular). Used as first URL segment.
+     * @param string $routeName  Prefix of the route name.
+     * @param string $controller Name of the current CrudController.
+     */
+    protected function setupFetchOperationRoutes($segment, $routeName, $controller)
+    {
+        preg_match_all('/(?<=^|;)fetch([^;]+?)(;|$)/', implode(';', get_class_methods($this)), $matches);
```

Comment:
Needs a comment:
```php
// get all method names on the current model that start with "fetch" (ex: fetchCategory)
// if a method that looks like that is present, it means we need to add the routes that fetch that entity 
```

Reply:
Done :+1: 

I am adding the comments in my local branch, it's not finished this PR.

Reply:
No, that's stupid. I expect ```preg_match()``` to be a lot faster than a ```foreach()``` with ```Str::startsWith()```. Let's favour speed over readability in this case. Bad @tabacitu ! Bad! 😆